resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -401,6 +401,9 @@ feedback.validation.no_sentiment=Please select how you're feeling.
 general.close_button=Close
 general.configure_button=Configure
 general.create_button=Create
+general.execution.cancelled=canceled
```

Comment:
Are you going to remove `cloud_debug.execution.cancelled` etc?

Reply:
They are still used in CloudDebugRunState